src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountForensicsSeriesBuilderTest.java

```diff
@@ -0,0 +1,51 @@
+package io.jenkins.plugins.forensics.miner;
+
+
+import java.util.Map;
+
+import io.jenkins.plugins.util.BuildAction;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+
+
+class RelativeCountForensicsSeriesBuilderTest {
+
+    @Test
+    void computeSeries() {
+        CommitStatistics commitStatistics = mock(CommitStatistics.class);
+        when(commitStatistics.getAuthorCount()).thenReturn(2);
+        when(commitStatistics.getFilesCount()).thenReturn(3);
+        when(commitStatistics.getCommitCount()).thenReturn(4);//
+        ForensicsBuildAction action = mock(ForensicsBuildAction.class);
+        when(action.getTotalLinesOfCode()).thenReturn(2);//
```

Comment:
Clean up your code before a commit. 

```suggestion
        when(action.getTotalLinesOfCode()).thenReturn(2);
```